### DISCUSSION TITLE
Peridigm conctact query

### DISCUSSION THREAD

**Discussion by deepakbiitk** [peridigm/peridigm#361] [General]
Hi all,

I was performing a contact simulation similar to the example impact test given in the examples. I refined the discretization of the plate. However, I forgot to change the search and contact radius in my input file. I saw contact force in the plate without any interaction with ball as shown below
![image](https://github.com/peridigm/peridigm/assets/51839991/18ad3031-0d1b-4999-aa35-fc3b4ad5e910)

Contact section in my file

![image](https://github.com/peridigm/peridigm/assets/51839991/d41ad76d-6ec3-45cd-b937-3c046ce41878)

It seems there is self contact happening between the particles of the plate. These forces go way if I reduce my contact radius and search radius. Is this something we expect during contact simulations?

Thanks,
Deepak

**Comment by djlittl** [comment]
There shouldn't be contact between nodes that are bonded to each other, this is disallowed in the code.  Is it possible that there are nodes in the plate that are not bonded to each other but are separated by a distance that is less than the contact radius?  This wouldn't normally happen if the horizon and the contact radius are set to sane values, for example horizon ~3x times the mesh spacing and the contact radius ~1.5 times the mesh spacing.  But in your example, maybe the contact radius is much larger than the horizon?

**Reply by deepakbiitk** [reply]
I created my mesh using Cubit. I took reference to journal files available in examples. This is the code I used

reset
transform mesh output scale 1e-3
#{PLATE_WIDTH= 80.0}
#{PLATE_THICKNESS = 2.0}
#{STRIKER_RADIUS = 7.0}
#{PLATE_MESH = PLATE_THICKNESS/6}
#{STRIKER_MESH = 2.0}

create brick width {PLATE_WIDTH} depth {PLATE_WIDTH} height {PLATE_THICKNESS}
create sphere radius {STRIKER_RADIUS}
move volume 2 x 0 y 0 z {STRIKER_RADIUS + OFFSET}
volume 1 size {PLATE_MESH}
mesh volume 1

volume 2 scheme tetmesh
volume 2 size {STRIKER_MESH}
mesh volume 2

block 1 volume 1
block 2 volume 2

nodeset 1 surface 3 4 5 6
nodeset 2 volume 2

export genesis 'drop_test.g' overwrite
